platform/features/conductor-v2/conductor/src/ui/TimeConductorMode.js

```diff
@@ -177,23 +177,61 @@ define(
          */
         TimeConductorMode.prototype.deltas = function (deltas) {
             if (arguments.length !== 0) {
-                var oldEnd = this.conductor.bounds().end;
-
-                if (this.dlts && this.dlts.end !== undefined) {
-                    //Calculate the previous raw end value (without delta)
-                    oldEnd = oldEnd - this.dlts.end;
+                var bounds = this.calculateBoundsFromDeltas(deltas);
+                this.dlts = deltas;
```

Comment:
is there any reason we can't just call this "deltas" instead of abbreviating it?